ehsm_kms_service/function.js

```diff
@@ -500,6 +504,7 @@ const _checkParams = function (req, res, next, nonce_database, DB) {
           return

```

Comment:
ReportControlTarget(req.ip, new Date().getTime()) at above line 499/503.

Reply:
ReportControlTarget(req.ip, new Date().getTime()) at above line 439/442.

Reply:
It will be fixed